rama-tcp/src/client/mod.rs

```diff
@@ -4,8 +4,9 @@
 pub mod service;
 
 mod connect;
+pub mod pool;
```

Comment:
do not squeeze it between here, there is plenty of space in file. Give it its own nice place ;)